Src/Couchbase.Linq/SaveOptions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Couchbase.Linq
+{
+    /// <summary>
+    /// Provides options controlling how changes are saved via <see cref="IBucketContext"/>.
+    /// </summary>
+    public class SaveOptions
```

Comment:
What if in the future, we wanted to have levels of consistency we wished to apply. For example, failing hard vs a warning if a consistency check failed....just an example. Are we future-proofing enough here?